src/looprestoration.rs

```diff
@@ -136,3 +137,160 @@ decl_looprestorationfilter_fns! {
     fn dav1d_wiener_filter7_16bpc_neon,
     fn dav1d_wiener_filter5_16bpc_neon,
 }
+
+// 256 * 1.5 + 3 + 3 = 390
+const REST_UNIT_STRIDE: usize = 390;
+
+// TODO Reuse p when no padding is needed (add and remove lpf pixels in p)
+// TODO Chroma only requires 2 rows of padding.
```

Comment:
Can you make these 2 from C doc comments?  I guess you can do that in the cleanup PR, though.  I usually add back comments in the cleanup PRs.

Reply:
Oh, for this one I *did* cleanup `padding` as much as I could. That's why these C comments are here.

In general I'd prefer not to make TODOs doc comments. They're not meant to be part of the generated docs, and so should be regular comments.

Reply:
Oh true.  For non TODO-like comments, those should be doc comments, right?

Reply:
Well, they should be doc comments if they're meant to be part of the function's rendered docs. But sometimes you want to leave comments that are more of an implementation detail than part of the function's public API. Mostly this comes up with `TODO` comments, but sometimes it's appropriate to leave e.g. a `NOTE` comment that you don't want to be part of the public docs.